src/js/client/errorBinder.js

```diff
@@ -0,0 +1,145 @@
+/*
+
+    These client side grades use model->view bindings like those used with `gpii.templates.binder` to associate
+    validation errors reported by the validator with onscreen elements.  That "binding" structure looks something like:
+    
+    ```
+    bindings: {
+        "key": {
+            selector: "selector1",
+            path:     "path1"
+        },
+        "selector2": "path2"
+    }
+    ```
+
+    The map of bindings used by the base component are stored under `options.errorBindings`.  By default, the component
+    tries to pick up the existing value from `options.bindings`, so that you can easily reuse existing bindings from
+    grades like 'templateFormControl`.
+
+    The core grade requires a `gpii-handlebars` `renderer` component.  An extended version of the `templateFormControl`
+    grade that performs all the necessary wiring is also included here.
+
+ */
+
+/* global fluid, $ */
+(function () {
+    "use strict";
+    var gpii = fluid.registerNamespace("gpii");
+    fluid.registerNamespace("gpii.schemas.client.errorBinder");
+
+    gpii.schemas.client.errorBinder.displayErrors = function (that) {
+        // Get rid of any previous validation errors.
+        fluid.each(that.locate("validationError"), function (element) {
```

Comment:
I believe that, by jQuery standard semantics, that.locate("validationError").remove() is sufficient here